docker/accounts/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM golang:1.15
+LABEL maintainer="NebulousLabs <devs@nebulous.tech>"
+
+ENV GOOS linux
+ENV GOARCH amd64
+
+WORKDIR /root
+
+RUN go get -d -u github.com/NebulousLabs/skynet-accounts/... && \
```

Comment:
We might want to change this to fetch the latest tag or the latest release or something like that. Otherwise it would be using the latest `master` which will not be always stable.